tests/unit/events/test_logging.py

```diff
@@ -24,3 +27,16 @@ def test_clears_preexisting_event_manager_state(self, mock_global_event_manager)
         setup_event_logger(flags=flags)
         assert len(manager.loggers) == 0
         assert len(manager.callbacks) == 0
+
+    def test_specify_max_bytes(
+        self,
+        mocker: MockerFixture,
+        mock_global_event_manager,
+    ) -> None:
+        patched_file_handler = mocker.patch("dbt_common.events.logger.RotatingFileHandler")
+        args = Namespace(log_file_max_bytes=1234567)
+        set_from_args(args, {})
+        setup_event_logger(get_flags())
```

Comment:
One theory I had was that although this commit primarily just _moves_ the test, I also made one small change which is highlighted here. Specifically, in this test previously, these methods were called via the module instead of directly. That is these two lines were
```python
flags.set_from_args(args, {})
setup_event_logger(flags.get_flags())
```
However modifying these lines and the the associated import did not solve the broken test issue we're seeing 🤔